<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="jajit-z01" ignored="0" name="Parallel Tests" tests="2" failures="2" timestamp="2021-08-18T17:26:52 IST" time="17.226" errors="0">
  <testcase name="CloseBrowser" time="0.014" classname="seleniumex.TestNG">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at seleniumex.TestNG.CloseBrowser(TestNG.java:19)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- CloseBrowser -->
  <testcase name="OpenBrowser" time="17.174" classname="seleniumex.TestNG">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [ToolsQA – Demo Website to Practice Automation – Demo Website to Practice Automation] but found [ToolsQA],
	expected [ToolsQA – Demo Website to Practice Automation – Demo Website to Practice Automation] but found [ToolsQA]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [ToolsQA – Demo Website to Practice Automation – Demo Website to Practice Automation] but found [ToolsQA],
expected [ToolsQA – Demo Website to Practice Automation – Demo Website to Practice Automation] but found [ToolsQA]
at seleniumex.TestNG.OpenBrowser(TestNG.java:40)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- OpenBrowser -->
</testsuite> <!-- Parallel Tests -->
